libs/base/ui/search/searchbox/src/searchbox.model.ts

```diff
@@ -1,9 +1,9 @@
 export interface SearchAttributes {
   open?: boolean;
   /**
-   * Provides floating behaviour on small screens
+   * Provides floating behavior
    */
-  xsFloated?: boolean;
+  float?: boolean;
```

Comment:
We are removing a property of public interface which is also a component attribute. I'd say it's a a breaking change. Can we think of backward compatible solution?